docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md

```diff
@@ -0,0 +1,39 @@
+# Existing EKS Cluster AWS Mixed Observability Accelerator
+
+## Objective
+
+This pattern depends on the previous deployment of [_Single New EKS Cluster Observability Accelerator_](./single-new-eks-cluster.md).
+After deploying this pattern you will have the same setup as in [_Single New EKS Cluster AWS Mixed Observability Accelerator_](./single-new-eks-mixed-observability.md).
+
+## Deploying
+
+1. Follow the instructions in [_Single New EKS Cluster Observability Accelerator_](./single-new-eks-cluster.md).
+
+2. Edit `~/.cdk.json` by setting the kubectl role name, as provided by the output of the above deployment of the _Single New EKS Cluster Observability Accelerator_.
+
+```json
+...
```

Comment:
you should also callout clusterName should be updated with your EKS Cluster Name